inputfiles/addedTypes.json

```diff
@@ -2207,6 +2207,20 @@
                         }
                     }
                 }
+            },
+            "MSLaunchUriCallback": {
+                "name": "MSLaunchUriCallback",
```

Comment:
Ah, this is a problem. Could you remove this and add `MSLaunchUriCallback` to "Window" field of knownTypes.json?

Reply:
Ok, sure.. will mve it to knownTypes.json...